simson/steel/steel_model.py

```diff
@@ -129,7 +129,10 @@ def extrapolate_trade_to_future(self, future_in_use_stock: Stock):
         product_demand = future_in_use_stock.inflow
         eol_products = future_in_use_stock.outflow
 
-        future_trade = {name: Parameter(name=name, dims=prm.dims.replace('h', self.dims['t']))
+        future_trade = {name: Parameter(name=name,
+                                        dims=prm.dims.replace('h', self.dims['t']).expand_by([self.dims['g']])
+                                            if 'scrap' in name
+                                            else prm.dims.replace('h', self.dims['t']))
```

Comment:
Maybe initialize `future_trade` as an empty dict and make this a loop with several lines. Seems a bit too long for a comprehension now...